swarm/api/config.go

```diff
@@ -48,12 +48,13 @@ type Config struct {
 	*network.HiveParams
 	Swap *swap.SwapParams
 	*network.SyncParams
-	Path      string
-	Port      string
-	PublicKey string
-	BzzKey    string
-	EnsRoot   common.Address
-	NetworkId uint64
+	Path       string
+	ListenAddr string
```

Comment:
so you need to add this to the config tests with the default value, it fails the CI test https://travis-ci.org/ethereum/go-ethereum/jobs/232519067#L1592